frontend/src/store.js

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    students: []
+  },
+  mutations: {
+    SET_STUDENTS(state, students) {
+      state.students = students
+    }
+  },
+  actions: {
+    getAllStudents( {commit} ) {
+      return fetch('http://localhost:3001/api/students')
```

Comment:
Now that we have the proxy, we can just do fetch('/api/...').

Reply:
Okay done!